db/migrate/20141003210742_add_docker_container_registry_url_to_providers.rb

```diff
@@ -2,7 +2,7 @@ class AddDockerContainerRegistryUrlToProviders < ActiveRecord::Migration
   def up
     add_column :katello_providers, :docker_registry_url, :string
     ::Katello::Provider.redhat.update_all(:docker_registry_url =>
-                                          Katello.config.redhat_docker_registry_url)
+                                          Katello.config.redhat_repository_url)
```

Comment:
Good point.. I guess if we are going to remove it any way no point in update all